### ISSUE TITLE
Version number in manual.pdf

### ISSUE THREAD

**Issue by akrabat** [rst2pdf/rst2pdf#919]
We need to get a handle on how the version number is created for manual.pdf.

On my computer, if I do:

```
git checkout 0.98
pip install -e .
cd doc
./gen_docs.sh
```

then I get `Version 0.98 (final)` on the manual.pdf

If @lornajane does this, she gets `Version 0.98 (dev124)`.

We need to determine how this works so that we can have consistency.

**Comment by lornajane** [comment]
I think it's a custom role, looking at https://github.com/rst2pdf/rst2pdf/commit/34d3044cb17ed8989f75750d9c9b3cd0ecfaa831

**Comment by stephenfin** [comment]
Yeah, this is a custom role. I'm not sure how it's getting `final`. I suspect you're building the docs outside of virtual environment, so it's using the system-wide install of `rst2pdf` rather than a more recent install from master

**Comment by akrabat** [comment]
My understanding is that the `final` one is the correct one for a checkout of tag `0.98`. The question is why Lorna didn't get that.
